docs/modules/ROOT/pages/references/dns-naming-scheme.adoc

```diff
@@ -0,0 +1,99 @@
+= DNS naming scheme
+
+NOTE: The DNS naming scheme fulfills the usability requirements documented in xref:appuio-cloud:ROOT:references/quality-requirements/usability/dns-naming-scheme.adoc[DNS Naming scheme].
+
+== Base domain
+
+The base domain is `appuio.cloud`.
+
+Common services that are zone independent are served on the third level.
+Services bound to a specific {zone}, will be served through the fourth level where the third level is the zone's identifier.
+Apps served through am {zone}'s default domain use the subdomain `apps` of a zone's base domain.
+
+.Known hosts and schemes of 'appuio.cloud'
+|===
+| Hostname | Description
+
+|`appuio.cloud`, `www.appuio.cloud`
+| Hosting the marketing homepage for {product}
+
+|`id.appuio.cloud`
+|identity provider for {product}
+
+|`console.appuio.cloud`
+| Frontend to manage cross zone concerns
+
+| `console.<zone>.appuio.cloud`
+| OpenShift console frontend of an {zone}
+
+| `api.<zone>.appuio.cloud`
+| Kubernetes API endpoint (e.g. for KubeConfig)
+
+| `cname.<zone>.appuio.cloud`
+| The DNS entry where users can point CNAME entries to
+
+| `<hostname>.apps.<zone>.appuio.cloud`
```

Comment:
`<app>.apps.<zone>.appuio.cloud`? because it's actually no host in a classical sense.